ui/app/scripts/analytics/views/dashboard.html

```diff
@@ -12,7 +31,7 @@
     </tr>
     </thead>
     <tbody>
-        <tr ng-repeat-start="item in items">
+        <tr ng-repeat-start="item in items | filter:filterQuery">
```

Comment:
I assume this filter is expected to be very primitive like any other filter operations we do in the UI.